### PR TITLE
finished code challenge

### PR DESCRIPTION
**Pull request by tahminaringer**

### CODE REVIEW

**Comment by tahminaringer** [comment]
- [x ] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ x] “Happy Path” - Expected outcome
     - [x ] Expected failure
     - [x ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [x ] Picture of whiteboard

File: javascript/code-challenges/graph/breadthFirst/breadth-first.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      } else {
+        visitedNodes.add(neighborNode);
+      }
+      queue.push(neighborNode);
+    }
+  }
+  return visitedNodes;
+}
+
+export mo
```
Comment (adrianhuebner) [inline javascript/code-challenges/graph/breadthFirst/breadth-first.js:31]: Code is looking good Tahmina! Once you have a chance to come back to this and add tests I will regrade for you!
